mono/mini/Makefile.am.in

```diff
@@ -1,6 +1,8 @@
 count=100000
 mtest=for_loop
 monodir=$(top_builddir)
+mono=$(if $(MONO_EXECUTABLE),$(MONO_EXECUTABLE),mono)
+sep=$(if $(LIST_SEPARATOR),$(LIST_SEPARATOR),:)
```

Comment:
Hmm, good question. We use glib for that, so whatever G_SEARCHPATH_SEPARATOR_S happens to be: https://github.com/mono/mono/blob/598062953a1ab9311f0562317cc93aeac120bc15/mono/metadata/assembly.c#L261